Feature/update home

jhossyd3v opened the pull request:
Include:
- Change of layout and content (assets and texts) for Hero, edit some font styles
- Change of asset for Testimonial and edit font styles 
- Change of Layout to receive References according to each page
- Add References in Home Page

Commit:
style: :bento: Update images for Home Page

Change assets for Hero and Testimonial in Home Page

Commit:
style: :lipstick: Update Main Hero

Change texts, images, update font styles and add styles for new elements

Commit:
style: :lipstick: Update styles for Testimonial component

Adjust width and position of the image, add padding-bottom to the section to gives space to extend enough the shape

Commit:
style: :lipstick: Fix font styles for Testimonial

Add line-height to quote paragraph, and add styles for a specific seletor for other parapgraphs to avoid overwrite quote paragraph styles

Commit:
style: :lipstick: Add image note in MainHero

Add the note in the corner right bottom of the image according to figma

Commit:
style: :construction: Change text on references component

Change text from lorem ipsum default text to references text in Home new design

Commit:
feat: :construction: Change how to Layout get References

Remove the property displayReferences and add an slot called references to receive the references with correct texts in each page

Commit:
feat: :construction: Add references in Home page

Edit Home Page implementation of layout to change the use of prop display reference for the slot references

HackMort reviewed (changes requested):
1. On mobile, the references text is breaking the layout because the link url is too long, we should fix this by using word-wrap, but pay attention up what amount of pixels do we need this, just to avoid weird issues later.
![image](https://user-images.githubusercontent.com/6466646/236054174-1d2228ac-a978-4415-af93-eb3fb96736eb.png)
2. Reduce the space between the image and the title/heading:
![image](https://user-images.githubusercontent.com/6466646/236054760-dc7b03a1-b834-4a70-b8db-8c8bc48cc1d8.png)
3. Increase the space at the bottom:
![image](https://user-images.githubusercontent.com/6466646/236054804-204bd3f4-8b0b-49be-8832-b48da2a3ef26.png)
4. Testimonial text seems to be a bit different, I played with the padding-right/left to 1.8rem and it looks a lot closer to the Figma.
![image](https://user-images.githubusercontent.com/6466646/236055438-a70ca7dc-728c-4903-8009-12fb140ec45b.png)
5. Reduce the padding-bottom on this section:
![image](https://user-images.githubusercontent.com/6466646/236055638-af29fd43-5b63-4c5e-a215-bb300acb9fc6.png)
6. Please make sure the content is aligned:
![image](https://user-images.githubusercontent.com/6466646/236055893-4a0ee37d-8245-4a7c-81c1-013a6350bbc8.png)
Looks like we have extra margins in the references component.
7. Starting at 1600px this happens, please fix!
![image](https://user-images.githubusercontent.com/6466646/236056190-62094e84-d807-4841-8bd8-e626f156a36b.png)

Commit:
style: :lipstick: Reduce space in Hero between image and title

Reduce padding-top of hero__content

Commit:
style: :lipstick: Fix styles for Main Hero

Add space between button and bottom of the Hero, fix the image on screens between 1000px and 1300px

Commit:
fix: :bug: Fix screen size for image change

Change from 100px to 1000 px the value in the source for change image, modify object-position for hero image to center from 1000px to 1200px

Commit:
style: :lipstick: Fix spaces and word-wrap for references

Reduce spaces between lets work section and references, Add word-wrap property in References section until 420px where link text fits well

Commit:
style: :lipstick: Add link HTML tag to link in Home references

Add link HTML tag to link in references in Home page, add styles according to Figma

Commit:
style: :lipstick: Edit styles for Testimonial component

Change padding left and right for quote and add padding bottom for component container , reduce space between testimonial and lets work section on mobile

Commit:
fix: :lipstick: Fix Main Hero for bigger screens

Remove the reduction of width for image that trigger the change of source in the picture in bigger screens, add a calc to keeps the image in the right border

Commit:
style: :lipstick: Fix shape of testimonial on bigger screens

Change width, height and bottom for testimonial shape to allow it to expand to borders, change padding bottom for testimonial container

Commit:
style: :lipstick: Fix font styles for References

Add custom font-size and line-height to p HTML tag in References to overwrite global styles for paragraphs

Commit:
style: :lipstick: Change the tag in References that has the prop wod-wrap

Add word-wrap to link HTML tag instead of  container

Commit:
feat: :lipstick: Changes of ISI in new design

Change texts, positions, spaces and font styles according to Figma

HackMort reviewed (approved):
Good job!
